### DISCUSSION TITLE
Wacom tablet flag/feature

### DISCUSSION THREAD

**Discussion by christowiz** [pilotmoon/Scroll-Reverser#124] [Ideas]
I've been looking for a way to separate the scrolling direction between my mouse and Wacom tablet.  I use the "unnatural" scrolling direction for my mouse scrolling but now my tablet's pan/scroll feature has reversed with the latest OS update.  I'm not sure if this is possible here but I thought I'd try.  My next step is to contact Wacom and ask for this new feature.

**Comment by pilotmoon** [comment]
Thanks for the suggestion. It may be possible - in fact older version of Scroll Reverser many years ago had this exact function but I removed it after Wacom driver supported scroll direction properly. 
I'll keep this suggestion in mind but I can't promise if I will add it again or if I do, when it will be.

**Reply by joostmaes** [reply]
Would really like this feature too. I also contacted Wacom to add this feature, but I don't expect much from them.

**Reply by mabam** [reply]
Same here. One year later and Wacom still didn’t fix this.
Would be great to have this feature back in Scroll Reverser!
